components/Internship/NextSteps.js

```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+function NextSteps(props) {
+    const { image, name, info } = props;
+    const [isFlipped,setFlipped] = useState(true);
+    return (
+        <div className="committee-card" onClick={() => setFlipped(!isFlipped)}>
```

Comment:
Since this is clickable, we'll want to somehow mark this as a clickable/tabbable element - either a `<button>` (which imo is maybe inappropriate here) or some sort of `role` and `aria` labels. Would encourage you to check out how some of the bigger players implement this!

Reply:
I think the common way is to say `role="button"` to mark it as clickable - so I did that. Will look into tab roles when I add tabbing!